### PR TITLE
Merge staging-v1.8.0-tasks into staging-v1.8.0

### PR DESCRIPTION
**Pull request by fernandofloresg**
##### SUMMARY

Part of the release process (#982) where we merge release tasks into the staging branch.

##### ISSUE TYPE

- Release Pull Request

### CODE REVIEW

**Review by rexemin — APPROVED** [review]
Looks good

**Review by ketankelkar — APPROVED** [review]
LGTM

**Review by AndreMarcel99 — APPROVED** [review]

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@
 Releases
 ========
 
-Version 1.8.0-beta.1
+Version 1.8.0
```
Comment (ddimatos) [inline docs/source/release_notes.rst:9]: I don't think this causes an RST issue, but the under line is longer than the text, I know when the underline is shorter than the text it causes an issue.

File: docs/source/release_notes.rst [hunk 2]
```diff
@@ -22,29 +22,46 @@ Minor Changes
...
 
 Bugfixes
 --------
 
-- ``zos_copy`` - Update option limit to include LIBRARY as dest_dataset/suboption value. Documentation updated to reflect this change.
-- ``zos_job_submit`` - Temporary files were created in tmp directory. Fix now ensures the deletion of files every time the module run.
-- ``zos_job_submit`` - The last line of the jcl was missing in the input. Fix now ensures the presence of the full input in job_submit.
+- ``zos_copy``
+
+    - Update option   to include `LIBRARY` as dest_dataset/suboption value. Documentation updated to reflect this change.
```
Comment (ddimatos) [inline docs/source/release_notes.rst:50]: There are some extra spaces, maybe it won't matter. 
```
 Update option   to include `LIBRARY` 
```

**Review by ddimatos — APPROVED** [review]
@fernandofloresg 2 minor comments which I don't think will impact anything, looks good thank you.

**Pull request merged**